docs/reference/setup/important-settings/discovery-settings.asciidoc

```diff
@@ -1,58 +1,56 @@
 [[discovery-settings]]
-=== Discovery settings
+=== Discovery and cluster formation settings
 
-Elasticsearch uses a custom discovery implementation called "Zen Discovery" for
-node-to-node clustering and master election. There are two important discovery
-settings that should be configured before going to production.
+There are two important discovery and cluster formation settings that should be
+configured before going to production so that nodes in the cluster can discover
+each other and elect a master node.
 
```

Comment:
I'd recommend defining discovery.zen.ping.unicast.hosts.resolve_timeout here too, unless there's already a reference page for it elsewhere.